toolbox/gpu_operator.py

```diff
@@ -57,13 +57,14 @@ def deploy_from_bundle(bundle):
         return PlaybookRun("gpu_operator_deploy_from_operatorhub", opts)
 
     @staticmethod
-    def deploy_from_operatorhub(version=None, channel=None):
+    def deploy_from_operatorhub(version=None, channel=None, installPlan="Manual"):
```

Comment:
installPlan is unused (need to pass it in the ansible params dict as `gpu_operator_installplan_approval`)